### ISSUE TITLE
make dist fails to deal with newlib symlinks

### ISSUE THREAD

**Issue by cpackham** [crosstool-ng/crosstool-ng#2012]
Just trying to prepare 1.26.0-rc1 and ran into this

```
$ make dist
/usr/bin/make  dist-xz dist-bzip2 am__post_remove_distdir='@:'
make[1]: Entering directory '/home/ct-ng/src/crosstool-ng'
/usr/bin/make  distdir-am
make[2]: Entering directory '/home/ct-ng/src/crosstool-ng'
if test -d "crosstool-ng-1.26.0_rc1"; then find "crosstool-ng-1.26.0_rc1" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "crosstool-ng-1.26.0_rc1" || { sleep 5 && rm -rf "crosstool-ng-1.26.0_rc1"; }; else :; fi
test -d "crosstool-ng-1.26.0_rc1" || mkdir "crosstool-ng-1.26.0_rc1"
cp: cannot overwrite directory 'crosstool-ng-1.26.0_rc1/packages/newlib-nano/4.2.0.20211231' with non-directory
cp: cannot overwrite directory 'crosstool-ng-1.26.0_rc1/packages/newlib-nano/4.3.0.20230120' with non-directory
cp: cannot overwrite directory 'crosstool-ng-1.26.0_rc1/packages/newlib-nano/3.3.0' with non-directory
cp: cannot overwrite directory 'crosstool-ng-1.26.0_rc1/packages/newlib-nano/4.1.0' with non-directory
make[2]: *** [Makefile:2672: distdir-am] Error 1
make[2]: Leaving directory '/home/chrisp/src/crosstool-ng'
make[1]: *** [Makefile:2667: distdir] Error 2
make[1]: Leaving directory '/home/chrisp/src/crosstool-ng'
make: *** [Makefile:2770: dist] Error 2
```

I assume this is a side effect of #1998 

@KirillSmirnov any thoughts on how to deal with this

**Comment by KirillSmirnov** [comment]
@cpackham Should be fixed in #2013

**Comment by cpackham** [comment]
All good. I was able to generate the 1.26.0-rc1 tarballs
